pages/index.vue

```diff
@@ -49,6 +49,7 @@
     </div>
 
     <div
+      id="what-we-do"
       class="relative bg-grey-000 pt-24 -mt-8 pb-20 px-4 sm:px-6 lg:pt-24 lg:pb-28 lg:px-8"
```

Comment:
we could do pb-4 and lg:pb-8 here to reduce the bottom padding

Reply:
Sure. Why don't we merge this and I'll let you give that a go as I'm up in bed now. ;)